### PR TITLE
27 create the main section of audio call page 2

### PR DESCRIPTION
**Pull request by jasonzhwang**
AC1:create AudioCall-practice page(when hovering button, go from light grey to icon grey)
Screenshot 2023-06-22 at 4 50 23 pm
AC2:create AudioCall-practice page(when hovering music player button, swap the background color with content color)
Screenshot 2023-06-22 at 4 50 32 pm
AC3:create AudioCall-Quiz page
Screenshot 2023-06-22 at 4 50 44 pm
AC4:create AudioCall-summary page
Screenshot 2023-06-22 at 4 50 52 pm

**new added components:**

1. IconButton
2. ImageButton
3. create "Home" "SprintGame" "AudioCallGame" folder to category components for better file structure

**Edited components:**

1. add outline Button feature into "NormalButton" component
2. change color categories for button component for clear illustration

**Extracted components:**

1. ScrollRow component extracted from TextArea, which is also used in Audicall-quiz row under the round coverImage 
3. Summary Frame extracted as one component which is used in audiocall summary and sprint summary page to hold two small dash boards
4. extracted mockData and store in the mockData file.
5. extracted DirectionGuide which is used in both Audiocall-practice and Audiocall-quiz components

**New Packages**

1. installed svgr package

### CODE REVIEW

File: src/components/AudioCallGame/AudioCallPractice/index.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <div className={styles.audioCall}>
+      <div onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave}>
+        <Button outline={false} variant="icon" size="normal" iconTypes="playerWithLabel" />
+      </div>
+      <img src="/fiveLikes.svg" alt="five likes" />
+      <div className={styles.wordListButton}>
+        {wordList.map((item, index) => (
+          <div key={index}>
+            <Button outline={true} label={item} color="greyIcon" size="normal" />
```
Comment (Double-Twelve) [inline src/components/AudioCallGame/AudioCallPractice/index.tsx:27]: this button's padding should be `14px, 13px, 14px, 13px`

File: src/components/AudioCallGame/AudioCallGame.module.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  > img {
+    margin: 39px auto 40px auto;
+  }
+  .wordListButton {
+    display: flex;
+    gap: 10px;
+  }
+  > button {
+    margin: 50px auto 45px auto;
+  }
```
Comment (Double-Twelve) [inline src/components/AudioCallGame/AudioCallGame.module.scss:24]: margin should be `margin: 40px auto;`

File: src/components/AudioCallGame/AudioCallGame.module.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+@import '../../styles/color.scss';
+
+.audioCall {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 100px auto 100px auto;
```
[the remark this answers is not in the document]
Comment (jasonzhwang) [reply src/components/AudioCallGame/AudioCallGame.module.scss]: Hi Double,

I have fixed the issues that you have been mentioned. Thanks.

File: src/layout/index.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { ReactNode } from 'react';
-import Header from '../components/Header';
+import Header from '../components/Home/Header';
 import Footer from './Footer';
```
Comment (tatoMa) [inline src/layout/index.tsx:3]: Why footer and header are in different levels of folders?

File: src/components/SprintGame/SprintSummary/SprintSummary.module.scss [hunk 5]
```diff
@@ -0,0 +1,4 @@
+.summaryDashGroup {
+  display: flex;
+  gap: 23.05px;
```
Comment (tatoMa) [inline src/components/SprintGame/SprintSummary/SprintSummary.module.scss:3]: No  .05 px in CSS. Just remove the decimal part.

File: src/components/AudioCallGame/AudioCallSummary/AudioCallSummary.module.scss [hunk 6]
```diff
@@ -0,0 +1,4 @@
+.summaryDashGroup {
+  display: flex;
+  gap: 23.05px;
```
Comment (tatoMa) [inline src/components/AudioCallGame/AudioCallSummary/AudioCallSummary.module.scss:3]: No  .05 px in CSS. Just remove the decimal part.

File: src/pages/sprint-dash.tsx
Comment (tatoMa) [inline src/pages/sprint-dash.tsx:1]: I don't get it why wrap a single component inside a page but nothing else?

Comment (jasonzhwang) [reply src/pages/sprint-dash.tsx:1]: Hi Boss, issues we have discussed have been fixed. Decimal pixels haven been edited, normal button name is changed to "RegularButton", p tag has been changed to span tag, please check. Thanks.

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**